src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -3,45 +3,98 @@ include::ROOT:partial$variables.adoc[]
 = Duplicate Detection
 
 {PRODUCT_NAME} through the duplicate detection feature helps maintain clean and accurate data by preventing record duplication.
-This is achieved by using rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.
+This is achieved by using rules to apply a criterion that determines how closely a field on a new or modified record matches the same field on an existing record.
 
-The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module, thereby preventing duplicates.
+The user interface displays a warning or an error message to alert the user of an existing record with  identical field values within a module, hence preventing the creation of duplicates.
 
-.Benefits of corteza duplicate detection:
-* Maintain data quality,
-* improves accuracy and efficiency in data,
-* enhances effective description of records,
-* optimizes data storage.
+.Benefits of using {PRODUCT_NAME} duplicate detection:
+* maintaining data quality,
+* Enhancing data accuracy and efficiency.
+* Enhancing effective description of records.
+* optimizes data storage
 
-== Validation Rules
+== Duplicate detection Rules
 
-The duplicate detection feature is enabled by using the value validation rules on a module.
-Validation rules verify that the designated fields in a record do not contain the same values, therefore ensuring that no duplicate values are saved when creating or editing a record.
+The duplicate detection feature is enabled by using validation duplicate detection rules on a module.
+The rules ensure that no duplicate values are recorded while creating or modifying a record by ensuring that the designated fields do not have the same values.
 
-Validation can be configured using two rules:
-
-* Strict duplicate value validation
+Duplicate detection validation can be configured using two rules:
 
+[cols="1s,5a"]
+|===
+| [#Strict-duplicate-validation]#<<Strict-duplicate-validation,Strict duplicate validation rule>>#
```

Comment:
please have these IDs all lower case (to be consistent)

`[#Strict-duplicate-validation]#<<Strict-duplicate-validation,Strict duplicate validation rule>>#` -> `[#strict-duplicate-validation]#<<strict-duplicate-validation,Strict duplicate validation rule>>#`

The same applies for the next one (soft validation)